src/stream/src/common/table/state_table.rs

```diff
@@ -869,12 +874,18 @@ where
     }
 
     fn insert_inner(&mut self, key: TableKey<Bytes>, value_bytes: Bytes) {
+        if crate::consistency::insane() && insane_random_drop() {
+            return;
+        }
         self.local_store
             .insert(key, value_bytes, None)
             .unwrap_or_else(|e| self.handle_mem_table_error(e));
     }
 
     fn delete_inner(&mut self, key: TableKey<Bytes>, value_bytes: Bytes) {
+        if crate::consistency::insane() && insane_random_drop() {
```

Comment:
Not very sure whether we need to muddle `delete_inner` and `update_inner` 🤔 TTL cases can be simulated as a false insert, but I can't find any cases for delete and updates

Reply:
For the storage bugs we found previously, deletes are lost. This can mock that case.

Reply:
Make senses to me.